tsconfig.json

```diff
@@ -7,8 +7,5 @@
   },
   "include": [
     "typescript/**/*.ts"
-  ],
-  "exclude":[
-    "typescript/test/test_application_ts/src/*.ts"
```

Comment:
i'm not sure the reason why we excluded the file in the tsconfig here, if it's safe to remove it?

Reply:
Because it's now outside of `typescript` folder (it's a fixture, so it's in fixtures).